### PR TITLE
Update my maintainer status for 1PW plugin

### PR DESCRIPTION
**Pull request by scottsb**
##### SUMMARY
I am no longer using Ansible regularly, so I don't want to receive notifications about changes. (I could also simply remove myself as a maintainer entirely if that's preferable.)

It doesn't likely matter much, but the email in my copyright line is also defunct, so I fixed that.

##### ISSUE TYPE
- Docs Pull Request

(not sure which issue type was right here)

##### COMPONENT NAME
onepassword

### CODE REVIEW

**Review by felixfontein — APPROVED** [review]
@scottsb thanks a lot for all your contributions to Ansible!

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -249,8 +249,10 @@ files:
     labels: onepassword
     maintainers: samdoran
   $lookups/onepassword.py:
+    ignore: scottsb
     maintainers: azenk scottsb
```
Comment (felixfontein) [inline .github/BOTMETA.yml:253]:
```suggestion
    maintainers: azenk
```

File: .github/BOTMETA.yml [hunk 2]
```diff
@@ -249,8 +249,10 @@ files:
...
   $lookups/onepassword_raw.py:
+    ignore: scottsb
     maintainers: azenk scottsb
```
Comment (felixfontein) [inline .github/BOTMETA.yml:256]:
```suggestion
    maintainers: azenk
```

**Review by felixfontein — COMMENTED** [review]
It's probably better to also remove you from `maintainers:`, in case the bot gets confused when you show up both in the maintainers list and in ignore ;-)

**Comment by samdoran** [comment]
@scottsb Thank you for your contributions. I learned some things from you that I still share with folks.

**Pull request merged**